Dockerfile

```diff
@@ -2,19 +2,20 @@ FROM mcr.microsoft.com/dotnet/sdk:6.0.301-bullseye-slim-amd64
 
 # install base software
 RUN mkdir -p /usr/share/man/man1 \
-    && mkdir -p /usr/share/man/man7 \
-    && apt-get update \
-    && apt-get install --no-install-recommends -y \
-    curl \
-    apt-transport-https \
-    gnupg \
-    wget \
-    dpkg \ 
-    zip \
-    default-jre \
-    && curl -fsSL https://deb.nodesource.com/setup_16.x | bash - \
-    && apt-get install --no-install-recommends -y nodejs \
-    && apt-get clean && rm -rf /var/lib/apt/lists/*
+  && mkdir -p /usr/share/man/man7 \
+  && apt-get update \
+  && apt-get install --no-install-recommends -y \
+  curl \
+  apt-transport-https \
+  gnupg \
+  wget \
+  dpkg \ 
+  zip \
+  default-jre \
+  && curl -fsSL https://deb.nodesource.com/setup_16.x | bash - \
+  && apt-get install --no-install-recommends -y nodejs \
+  && apt-get install -y --no-install-recommends libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb procps\
```

Comment:
Feels kinda weird to see libgtk dependency in a BE project but I suppose it's a problem that Cypress should be resolving.